cli/command/container/hijack.go

```diff
@@ -19,41 +19,40 @@ import (
 // TODO: This could be moved to `pkg/term`.
 var defaultEscapeKeys = []byte{16, 17}
 
-// A hijackedIOStreamer handles copying input to and output from streams to the
+// HijackOptions handles copying input to and output from streams to the
 // connection.
-type hijackedIOStreamer struct {
-	streams      command.Streams
-	inputStream  io.ReadCloser
-	outputStream io.Writer
-	errorStream  io.Writer
-
-	resp types.HijackedResponse
+type HijackOptions struct {
+	resp       types.HijackedResponse
+	ttyInput   io.ReadCloser
```

Comment:
I don't think we necessarily need this field either. Check to see if we actually ever call `Close()` on this value. We might be able to get away removing it the same you did with `outputStream` and `errorStream` and just use the passed in `streams.In()`.

Reply:
@jlhawn I saw this io.ReadCloser is tricker than the other two. On [line 104](https://github.com/AntaresS/cli/blob/64f436f08c10656745ce8c489d1f218b3a6f7690/cli/command/container/hijack.go#L104) it is being overwritten and `Close()` is called on [line 202](https://github.com/AntaresS/cli/blob/64f436f08c10656745ce8c489d1f218b3a6f7690/cli/command/container/hijack.go#L202). Seems like it is no longer an on/off switch any more like `outputStream` and `errorStream`

Reply:
The Close() on line 202 is effectively calling `streams.In().Close()` though.